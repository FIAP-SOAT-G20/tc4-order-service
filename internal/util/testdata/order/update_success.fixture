{
    "customer_id": 5,
    "status": "PENDING"
}
