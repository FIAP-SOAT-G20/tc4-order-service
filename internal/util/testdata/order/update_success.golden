{
    "id": 15,
    "customer_id": 5,
    "total_bill": "0.00",
    "status": "PENDING",
    "customer": {
        "id": 5,
        "name": "Test Customer 5",
        "email": "test.customer.5@email.com",
        "cpf": "321.654.987-00",
        "created_at": "2025-02-28T16:28:18Z",
        "updated_at": "2025-02-28T16:28:18Z"
    },
    "created_at": "2025-03-06T15:19:09Z",
    "updated_at": "2025-03-06T15:21:03Z"
}
